lua/laravel/_config.lua

```diff
@@ -31,6 +31,14 @@ local config = {
     ["websockets"] = "persist",
     ["queue:restart"] = "watch",
   },
+  environment = {
+    resolver = require "laravel.environment.resolver"(true, true, nil),
+    environments = {
+      ["local"] = require("laravel.environment.native").setup(),
```

Comment:
My idea with this is that you can easily overwrite the configuration calling the function. This makes if I add from the suported a new field you don't need to modify your local setup since the function is the one being executed.
Not in this file you will not pass variables but in your call to the `require("laravel").setup()` there you can set it.